internal/jimm/permissions/relations.go

```diff
@@ -250,3 +251,15 @@ func (j *permissionManager) parseTuple(ctx context.Context, tuple apiparams.Rela
 
 	return &t, nil
 }
+
+// RemoveModel removes all tuples that contain a model as the target.
+func (j *permissionManager) RemoveModel(ctx context.Context, model names.ModelTag) error {
+	const op = errors.Op("jimm.RemoveModel")
+	if model.Id() == "" {
+		return errors.E(op, errors.CodeBadRequest, "model tag cannot be empty")
+	}
+	if err := j.authSvc.RemoveModel(ctx, model); err != nil {
```

Comment:
Reading this I have some thoughts,
- If we keep this pattern, we'll need to think about the roles and responsibilities between this permissions manager and the authSvc.
- If other packages in `internal/jimm` are supposed to use the permissions manager instead of using the `authSvc` directly, what we accomplishing? If we compare it to the database, all the managers have a DB object to go directly to the DB.

I think I would drop this change and just use the authSvc directly in the juju manager to keep things simple. But open to different ideas on this.